README.md

```diff
@@ -131,10 +131,15 @@ TODO:
 
 ### Rendering
 
+Like iframes, portals can render their contents inline in another document. To ensure a smooth transition when activation occurs, and to limit the avenues for communication between the two documents, rendering generally occurs in the same way as it will when the portal is activated. This means that `document.visibilityState` and `document.hidden` will, like iframes, match the values in their host browsing context, even if they are offscreen. Similarly, `IntersectionObserver` will report intersections up to the portal contents viewport, but will assume that viewport is fully visible. Other behavior that depends on intersection with the viewport, such as lazy-loading images, behaves similarly.
+
+`requestAnimationFrame` should issue vsync callbacks as it would in the host document (in particular the host document should not control the frequency of animation updates), except that for performance reasons user agents may suspend or throttle callbacks to offscreen portals if the two documents are same-origin.
```

Comment:
I'd prefer "will" to "should" here; should makes it sound somewhat optional.

Reply:
Changed to simply "issues". The other "should" is trying to describe the purpose rather than the solution, but I'm open to changing it if you prefer other wording.